utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx

```diff
@@ -33,10 +62,55 @@ export function CardDocumentation() {
           </ul>
         )}
       />
+      <PreCodeForCodeString
+        className="gray-block mt-spacing"
+        codeRaw={`
+          <div class="card p-spacing-xl">
+            <div class="card__title">Title</div>
+            <p>Sed ut perspiciatis unde omnis iste natus error.</p>
+            <div class="flex justify-end">
+              <button type="button" class="button">Contact Us</button>
+              <button 
+                type="button" 
+                class="button button--solid ml-spacing"
+              >
+                See More
+                <span class="button--icon button--icon-right">
+                  <span
+                    class="utds-icon-before-arrow-right"
+                    aria-hidden="true"
+                  ></span>
+                </span>
+              </button>
+            </div>
+          </div>
+        `}
+      />
 
       <StaticExample
         title="Action Cards"
-        renderedExample={<LightBox image={actionCardsScreenshot} alt="Action Cards" className="flex-3up-gap" />}
+        renderedExample={(
+          <div className="flex flex-col gap">
```

Comment:
Should the cursor be a hand when these are hovered?

Reply:
Yes